setup.py

```diff
@@ -62,7 +62,7 @@
 
 install_requires = [
     'Flask-Admin>=1.3.0',
-    'Flask-Login<0.3.0,>=0.2.11',
+    'Flask-Login>=0.3.2',
```

Comment:
Flask-Security has not yet been released with support for Flask-Login 0.3.0+ so we need to keep the ``'Flask-Login<0.3.0,>=0.2.11',`` still.